app/src/main/java/com/droidhats/campuscompass/MainActivity.kt

```diff
@@ -88,4 +98,34 @@ class MainActivity : AppCompatActivity() {
             }
         }
     }
+
+    override fun onCreateOptionsMenu(menu: Menu): Boolean {
+        // Inflate the menu; this adds items to the action bar if it is present.
+        menuInflater.inflate(R.menu.nav_drawer_main_menu, menu)
+        return true
+    }
+
+    override fun onNavigationItemSelected(item: MenuItem): Boolean {
+
+        val navHostFragment: Fragment? = supportFragmentManager.primaryNavigationFragment
+        val currentFragment: Fragment? = navHostFragment?.getChildFragmentManager()?.getFragments()?.get(0)
```

Comment:
Please replace this line: `val currentFragment: Fragment? = navHostFragment?.getChildFragmentManager()?.getFragments()?.get(0)`

With: `val currentFragment: Fragment? = navHostFragment?.childFragmentManager?.fragments?.get(0)`

since we are calling it as a getter method call